### PR TITLE
[fre] Adds French covering grammar

### PR DESCRIPTION
**Pull request by ajmalanoski**
- [x] Updated `Unreleased` in `CHANGELOG.md` to reflect the changes in code or data.

I also took the liberty of organizing the French phonelist.

### CODE REVIEW

File: data/covering_grammar/tsv/fre_latn_phonemic.tsv [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+ti	ti
+ti	tj
+ti	si
+ti	sj
+um	œ̃
+um	ɔm
+un	œ̃
+ym	ɛ̃
+ym	im
+yn	ɛ̃
```
Comment (kylebgorman) [inline data/covering_grammar/tsv/fre_latn_phonemic.tsv:231]: Can you add a newline here?

File: data/phones/phones/fre_phonemic.phones [hunk 2]
```diff
@@ -1,42 +1,53 @@
...
+# *les uns les autres* /le.z‿œ̃ le.z‿otʁ/ 'each other (pl)' or 
+# *d'habitude* /d‿a.bi.tyd/ 'usually.' Strictly speaking, it should not be
+# included in a phonemic transcription, since syllabification is not part of
+# the underlying form. However, for many entries, the only transcription given
+# includes tie bars, so the tie bar is necessary if we don't want to filter 
+# them out. Additionally, the tie bar is used when indicating the 
+# pronunciation of a normally silent consonant in liaison, as in 
+# "grand" /ɡʁɑ̃.t‿/.
 ‿
-
```
Comment (kylebgorman) [inline data/phones/phones/fre_phonemic.phones:42]: You deleted the newline here, is that intentional? Something your text editor is doing for you automatically?

**Review by kylebgorman — APPROVED** [review]
Great, thanks.

File: data/phones/phones/fre_phonemic.phones (hunk 2 above)
Comment (ajmalanoski) [reply data/phones/phones/fre_phonemic.phones:42]: Huh, turns out it was my text editor doing that. Good to know.

**Pull request merged**